p2p/protocol.go

```diff
@@ -78,9 +79,15 @@ func (cap Cap) String() string {
 }
 
 // Less defines the canonical sorting order of capabilities.
-func (cap Cap) Less(other Cap) bool {
+func (cap Cap) Less(other Cap) int {
```

Comment:
```suggestion
func (cap Cap) Cmp(other Cap) int {
```
Wouldn't that be more correct? Adds more fallout, sorry ...

Reply:
Ah yes, indeed